src/index.html

```diff
@@ -4,9 +4,413 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__image">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="MOYO"
+            class="header__logo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--is-active is-active" href="#apple">APPLE</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">SAMSUNG</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#phones">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">ГАДЖЕТЫ</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">ПЛАНШЕТЫ</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photos">ФОТО</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#videos">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="catalog">
+        <article class="product-card catalog__item" data-qa="card">
```

Comment:
`<article>` isn't the best choice here
```suggestion
        <div class="product-card catalog__item" data-qa="card">
```